index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio | Home</title>
+    <!-- Google Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap"
+        rel="stylesheet">
+    <!-- Boxicon -->
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <!-- CSS -->
+    <link rel="stylesheet" href="styles.css">
+
+<body>
+    <!-- Start of Header Container -->
+    <header>
+        <nav class="navbar">
+          <div class="container flex">
+            <a href="index.html" class="logo">Evans</a>
+            <ul class="menu">
+              <li><a href="#portfolio">Portfolio</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
+            </ul>
+            <i class='bx bx-menu'></i>
+          </div>
+        </nav>
+    
+        <!-- start of banner -->
+        <section class="banner">
+          <div class="container">
+            <div class="banner-text">
+              <h1>I’m Nyamekye. <br>
+                Glad to see you!</h1>
```

Comment:
- [ ] Please fix this design below, we are expecting the `YOU!` to move to the red box position
- [ ] As for the bending line, please adjust your border radius at the bottom left position to match the original design
<img width="876" alt="1" src="https://github.com/evansnyamekye/Post-index/assets/77843050/e67b1e2e-73a8-4011-b7db-81168652ad3d">